frameworks/g-cloud-9/questions/services/standardsCSASTAR.yml

```diff
@@ -2,7 +2,7 @@ name: CSA STAR certification
 question: >
   Do you have a current CSA Security, Trust & Assurance Registry (STAR) certification that covers the security of
   your service?
-
+filter_label: CSA STAR
```

Comment:
Is Roz and/or Karl OK with this acronym? We haven't presented it unexplained before and need to make sure we have their input.